src/Microsoft.PowerShell.Commands.Utility/commands/utility/new-object.cs

```diff
@@ -465,7 +474,7 @@ private object CreateComObject()
                 return null;
             }
         }
-
+#endif
         #endregion Com
```

Comment:
I think the entire "com" region here should be under the #if !UNIX.

Reply:
Fixed.